.github/fabricbot.json

```diff
@@ -1616,17 +1616,28 @@
         "conditions": {
           "operator": "and",
           "operands": [
-            {
-              "name": "prTargetsBranch",
-              "parameters": {
-                "branchName": "dev17.3-preview2"
-              }
-            },
             {
               "name": "isAction",
               "parameters": {
                 "action": "merged"
               }
+            },
+            {
+              "operator": "or",
+              "operands": [
+                {
+                  "name": "prTargetsBranch",
+                  "parameters": {
+                    "branchName": "dev17.3.x"
+                  }
+                },
+                {
+                  "name": "prTargetsBranch",
+                  "parameters": {
+                    "branchName": "dev17.3-preview2"
+                  }
+                }
+              ]
```

Comment:
This change associates both the dev17.3-preview2 and dev17.3.x branches with the 17.3 milestone. This is a little wonky because we don't really care about tracking the dev17.3-preview2 branch anymore, but I was using a tool to update this file and this minimized the churn.